### ISSUE TITLE
Create a team for libc++ label

### ISSUE THREAD

**Issue by mordante** [llvm/llvm-project#53352]
Please add a label for https://github.com/llvm/llvm-project/labels/libc%2B%2B

**Comment by EugeneZelenko** [comment]
Did you mean `team`?

**Comment by asl** [comment]
Apparently we cannot as github strips the special symbols and therefore the name clashes with "libc" @tstellar ?

**Comment by tstellar** [comment]
@asl That's strange, because we have both issue-subscribers-c and issue-subscriber-c++ teams, which in theory would be the exact same problem.

**Comment by asl** [comment]
Yes, I know. But please try to create libc++ team :)

**Comment by tstellar** [comment]
@asl Yeah, I already tried :)  I still don't understand why it worked for c and c++ but not libc and libc++ ?  Maybe it would work if libc++ was created first?  I added a libcxx team, so one option would be to add special case for the libc++ tag in the GitHub action and map it to libcxx.
